### ISSUE TITLE
GPL-882 - As a reefer user I would like to be able to scan boxes into shelves without persisting the moves [Heron]

### ISSUE THREAD

**Issue by stevieing** [sanger/labwhere#323]
**User story**
As a reefer user I would like to be able to scan boxes into shelves without persisting the moves

priority: 2

**Who are the primary contacts for this story**
Bart

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] I would be able to select a reefer which would take me to a page
- [ ] There would be a reefer page which would be a visual representation of the reefer
- [ ] This page would be made up of 14 racks and 3 shelves. Each shelf can contain 6 boxes
- [ ] I would be able to scan a box into one of the shelves
- [ ] I would be able to count the number of boxes on each shelf and enter that number into the shelf on the page
- [ ] I would be able to save this information without persisting it into the Labwhere database
- [ ] I would be able to retrieve this information at a later date

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by rl15** [comment]
Bart Siwek wrote (Monday, 15 March 2021 at 15:27)

Steve Inglis is working on ‘virtual’ reefer layout that will help with scanning boxes into reefers and GPL-882 & GPL-883 are a part of this improvement

**Comment by stevieing** [comment]
Closed as no longer needed.
